rcl_lifecycle/QUALITY_DECLARATION.md

```diff
@@ -122,31 +138,37 @@ It also has several test dependencies, which do not affect the resulting quality
 
 `lifecycle_msgs` provides message and services for managing lifecycle nodes.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/rcl_interfaces/blob/master/lifecycle_msgs/QUALITY_DECLARATION.md).
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcl_interfaces/blob/foxy/lifecycle_msgs/QUALITY_DECLARATION.md).
 
 #### `rcl`
 
 `rcl` is the ROS 2 client library in C.
 
-It is **Quality Level 4**, see its [Quality Declaration document](../rcl/QUALITY_DECLARATION).
+It is **Quality Level 2**, see its [Quality Declaration document](../rcl/QUALITY_DECLARATION).
 
 #### `rcutils`
 
 `rcutils` provides commonly used functionality in C.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/rcutils/blob/master/QUALITY_DECLARATION.md).
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcutils/blob/foxy/QUALITY_DECLARATION.md).
 
 #### `rmw`
 
 `rmw` is the ROS 2 middleware library.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/rmw/blob/master/rmw/QUALITY_DECLARATION.md).
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rmw/blob/foxy/rmw/QUALITY_DECLARATION.md).
 
 #### `rosidl_runtime_c`
 
 `rosidl_runtime_c` provides runtime functionality for rosidl message and service interfaces.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/rosidl/blob/master/rosidl_runtime_c/QUALITY_DECLARATION.md).
+It is **Quality Level 1**, see its [Quality Declaration document](https://github.com/ros2/rosidl/blob/foxy/rosidl_runtime_c/QUALITY_DECLARATION.md).
+
+#### `tracetools`
+
+The `tracetools` package provides utilities for instrumenting the code in `rcl_lifecycle` so that it may be traced for debugging and performance analysis.
+
+It is **Quality Level 1**, see its [Quality Declaration document](https://gitlab.com/ros-tracing/ros2_tracing/-/blob/foxy/tracetools/QUALITY_DECLARATION.md).
```

Comment:
if it matters, `rcl_lifecycle` @ foxy doesn't depend on `tracetools` (it only does in rolling)